sql/mob_groups.sql

```diff
@@ -13519,7 +13513,7 @@ INSERT INTO `mob_groups` VALUES (28,5526,272,'Slinking_Slug',0,0,0,0,0,121,123,0
 INSERT INTO `mob_groups` VALUES (29,5527,272,'Bloodspattered_Fly',0,0,0,0,0,120,123,0);
 INSERT INTO `mob_groups` VALUES (30,2671,272,'Mineral_Eater',0,0,0,0,9999,120,123,0);
 INSERT INTO `mob_groups` VALUES (31,3102,272,'Passage_Crab',0,0,0,0,0,120,123,0);
-INSERT INTO `mob_groups` VALUES (32,4980,272,'Unyielding_Tarichuk',0,0,0,0,0,120,123,0);
+INSERT INTO `mob_groups` VALUES (32,5639,272,'Irascible_Tarichuk',0,0,0,0,0,120,123,0);
```

Comment:
Others: If you see a name change in this diff, it's because there's currently two groups with the same group name in a zone. Wren is updating the name of the duplicate group to match the mobs that are already in that group.

GitHub might refuse to properly expand surrounding lines, but in zone 272, there were two batches of Tarichuk: group 14 and group 32. The tarichuks in the zone are already divided into these separate groups, Group 32 in mob_spawn_points are already named Irascible Tarichuk. So Wren's updating the name in mob_groups to match the mobs belonging to it, and giving them a new mob_pool.

Reply:
The reason this happened, btw, is because when the name column was added to the groups table, it was infilled from the pools table for whatever poolid was currently in the row, rather than from the mob_spawn_points table.